README.md

```diff
@@ -17,14 +17,16 @@ npm install
 Then build the extension:
 
 ```shell
-npm run build
+npm run build:firefox
+# or
+npm run build:chrome
```

Comment:
I just remembered that the `start:*` scripts start by running `build:*`, so technically this step is no longer needed.

Reply:
Won't we still need it when we would actually produce build artifacts to publish releases?

Reply:
I was referring to the development environment instructions in the README.md. Because `start:firefox` runs `build:firefox`, the developer doesn't need to do `build:firefox` before calling `start:firefox`.

The `build:production` script will continue to exist, that's the one we'd run on CI.